src/main/java/org/project36/qualopt/web/rest/StudyResource.java

```diff
@@ -127,6 +128,21 @@ public StudyResource(StudyRepository studyRepository, StudyService studyService,
         return ResponseUtil.wrapOrNotFound(Optional.ofNullable(study));
     }
 
+    /**
+     * GET  /studies/info/:id : get the study info with the specified id.
+     *
+     * @param id the id of the study, from which the study info will be retrieved
+     * @return the ResponseEntity with status 200 (OK) and with body the study, or with status 404 (Not Found)
+     */
+    @GetMapping("/studies/info/{id}")
+    @Timed
+    public ResponseEntity<StudyInfoDTO> getStudyInfo(@PathVariable Long id) {
+        log.debug("REST request to get study info: {}", id);
+        Study study = studyRepository.findOneWithEagerRelationships(id);
+        StudyInfoDTO studyInfo = new StudyInfoDTO(study);
+        return new ResponseEntity<>(studyInfo, HttpStatus.OK);
```

Comment:
Maybe as a precaution, add a case for when the study object was not found. Like returning a ResponseEntity object with a different status code.

Reply:
Yeap definitely. Thanks